packages/suite-desktop/src/support/screens/TorLoadingScreen.tsx

```diff
@@ -189,42 +188,23 @@ export const TorLoadingScreen = ({ callback }: TorLoadingScreenProps) => {
                                 isRed={torStatus === TorStatus.Error}
                                 value={torStatus === TorStatus.Error ? 100 : progress}
                             />
+
                             <ProgressMessage>
                                 {torStatus === TorStatus.Error ? (
-                                    <P>Failed</P>
+                                    <span>Failed</span>
```

Comment:
Can we make it translatable?

Reply:
I mean the whole TorLoadingScreen. I know it's not a part of this PR... But maybe we could just wrap it by `IntlProvider` a provide the language somehow? 🤔

Reply:
600b02475b6bd2834b30af81f5acccc9724fc91c